src/mumble/mumble.pro

```diff
@@ -143,7 +143,8 @@ HEADERS *= BanEditor.h \
     PathListWidget.h \
     XMLTools.h \
     Screen.h \
-    SvgIcon.h
+    SvgIcon.h \
+	Markdown.h
```

Comment:
The file uses spaces for indentation, but the line you wrote is indented with tabs.

Honestly, it's not an issue because we're going to switch to CMake soon, but if you're going to push the suggested changes I would fix this as well.

Reply:
Yeah I'll fix it. I made this mistake every single time I edit one of these files :facepalm:

Reply:
No worries.

Consider that we have mixed indentation across all `.pro` and `.pri` files.